src/test/java/Pages/CreateAccountPage.java

```diff
@@ -0,0 +1,119 @@
+package Pages;
+
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+public class CreateAccountPage extends BasePage{
```

Comment:
Page object is fully incorrect:
1. Assertions
2. Senseless granular methods instead of 1 method `createAccount(User user)`